doc/DeveloperGuide.md

```diff
@@ -59,6 +59,7 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | user | add a new person |
 `* * *` | user | delete a person | remove entries that I no longer need
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
+`* * *` | user | to be able to edit person particulars | update the particulars without deleting and re-adding
```

Comment:
This is not part of W6.6h. Ensure that the PR is clean next time round (i.e. only PR the changes that pertain to it). :)